go.mod

```diff
@@ -1,21 +1,16 @@
 module github.com/btcsuite/btcd
 
 require (
-	github.com/aead/siphash v1.0.1 // indirect
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
+	github.com/btcsuite/btcutil v1.0.2
 	github.com/btcsuite/go-socks v0.0.0-20170105172521-4720035b7bfd
-	github.com/btcsuite/goleveldb v0.0.0-20160330041536-7834afc9e8cd
-	github.com/btcsuite/snappy-go v0.0.0-20151229074030-0bdef8d06723 // indirect
+	github.com/btcsuite/goleveldb v1.0.0
```

Comment:
Given modules exist now, do we still need to vendor `goleveldb`?

Reply:
Our fork looks to be a year or so behind mainline: https://github.com/syndtr/goleveldb